yarn-completion.bash

```diff
@@ -1015,14 +1015,17 @@ _yarn_yarn() {
 }
 
 _yarn() {
-	# shellcheck disable=SC2064
-	trap "
-		PWD=$PWD
-		$(shopt -p extglob)
-		set +o pipefail
-	" RETURN
-
-	shopt -s extglob
+	if [[ "$SHELL" != "/bin/zsh" && "$SHELL" != "/usr/bin/zsh" ]]; then
```

Comment:
What if zsh is installed in a non-standard path? Like if installed by homebrew or something? Is there a more reliable way of determining this?

Or perhaps instead of making this a check of the negative ("this shell is not zsh"), maybe it would be better to make this a check of the affirmative ("this shell is bash").